conda/install.py

```diff
@@ -78,14 +74,27 @@ def translation(found_path):
             return root_prefix + "/" + found
         return re.sub(path_re, translation, path).replace(";/", ":/")
 
+    def url_path(path):
+        path = abspath(path)
+        if sys.platform == 'win32':
+            path = '/' + path.replace(':', '|').replace('\\', '/')
+        return 'file://%s' % path
+
 # Make sure the script stays standalone for the installer
 try:
-    from conda.config import remove_binstar_tokens
+    from conda.config import remove_binstar_tokens, pkgs_dirs, url_channel
```

Comment:
@ilanschnell, when testing this with `constructor`, I found that this would fail as soon as `conda` was installed because it can't yet see a `yaml` parser. Catching the `ImportError` exception isn't enough, because the YAML parser loader doesn't _raise_ an `ImportError` in that case; rather, it calls `sys.exit()`. I could catch `SystemExit` too, but it seems simpler just to combine all of the `conda.` imports into a single exception block.

In fact, wouldn't it be better still to import only if `__name__ != 'main'`?